javalib/src/main/scala/java/lang/Thread.scala

```diff
@@ -24,8 +24,9 @@ class Thread private (runnable: Runnable) extends Runnable {
   final def getName(): String =
     this.name
 
-  @stub
-  def getStackTrace(): Array[StackTraceElement] = ???
+  // Stub implementation which is more useful than '???'.
```

Comment:
I think this comment should be removed, or at least truncated to "Stub implementation". The part "which is more useful than '???'" is only meaningful because the previous code had `???`. But once this is merged, it won't make sense anymore.

Reply:
Done